.claude/skills/workflow/scripts/Sync-SessionDocumentation.ps1

```
@@ -0,0 +1,134 @@
+
+Write-Host "📝 Syncing session documentation: $SessionLogPath" -ForegroundColor Cyan
+
+# Step 1: Query agent history
+Write-Host "  [1/5] Querying agent history" -ForegroundColor Yellow
+$historyResult = Invoke-AgentOrchestrationMCP -ToolName 'agents://history' -Arguments @{}
+if ($historyResult.Fallback) {
+    Write-Warning "Agent history unavailable — generating skeleton documentation."
+}
+
+# Step 2: Generate Mermaid sequence diagram
+Write-Host "  [2/5] Generating workflow sequence diagram" -ForegroundColor Yellow
```

PSScriptAnalyzer (1.24.0)

[warning] File 'Sync-SessionDocumentation.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)